### ISSUE TITLE
AccessibilityState not working as expected on View

### ISSUE THREAD

**Issue by jagorrin** [microsoft/react-native-windows#4368]
### Environment

If you are using latest version:
1. `react-native -v`: 0.60.6
2. `react-native run-windows --info`:
  System:
    OS: Windows 10 10.0.18363
    CPU: (12) x64 Intel(R) Core(TM) i7-8700K CPU @ 3.70GHz
    Memory: 18.01 GB / 31.84 GB
  Binaries:
    Node: 10.15.0 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.21.1 - C:\Program Files (x86)\Yarn\bin\yarn.CMD
    npm: 6.13.7 - C:\Program Files\nodejs\npm.CMD
  npmPackages:
    react: 16.8.6 => 16.8.6
    react-native: 0.60.6 => 0.60.6
    react-native-windows: 0.60.0-vnext.149 => 0.60.0-vnext.149
    rnpm-plugin-windows: 0.3.5 => 0.3.5
  Installed UWP SDKs:
    10.0.14393.0
    10.0.15063.0
    10.0.16299.0
    10.0.17134.0
    10.0.17749.0
    10.0.17763.0
    10.0.18362.0

Then, specify:

- Target Platform Version(s): 10.0.18362.0

- Target Device(s): Desktop

- Visual Studio Version: 2017

- Build Configuration: Debug

### Steps to Reproduce

Create a project using react-native-windows that includes a View on which an accessibilityState exists.

In my specific scenario, I have a component that is a checkbox, which is built on top of the RNW View component. I provide it an accessibilityRole of "checkbox" and an accessibilityState that has the value of "checked" as either true or false depending on if it is checked. I am not using the RNW CheckBox component due to issues outlined in #4367.

### Expected Behavior

When using Narrator, the accessibility state read aloud should be accurate.

### Actual Behavior

The accessibility state is always being read as "unchecked" even when "checked" is provided.

### Code Example
This code produces the screenshot above.

```
import React, { PureComponent } from "react";
import { View } from "react-native";
...
const accState = { checked: this.state.checked ? true : false };
<View accessibilityRole={"checkbox"} accessibilityState={accState}>
...
</View>
...
```

**Comment by NickGerleman** [comment]
Also tracking with #4042. We need to update our view managers to move from the deprecated `accessibilityStates` (plural) API, which will be completely removed in 0.62.

**Comment by NickGerleman** [comment]
I think this is relatively easy to do FWIW.

**Comment by NickGerleman** [comment]
Oh wait a second. @jagorrin `accessibilityState` was added as a property in `0.61` and you seem to be running `0.60.6`. We haven't actually implemented `accessibilityState` yet in `0.61` either, but for `0.60` you need to use the plural `accessibilityStates` (which will be removed in two versions).

**Comment by jagorrin** [comment]
@NickGerleman Thanks for the info. I changed from using _accessibilityState_ to _accessibilityStates_ and now it works as expected. Closing this issue.
